corehq/apps/app_manager/management/commands/cmitfb_identify_broken_forms.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from datetime import datetime
+from django.core.management import BaseCommand
+from django.http import Http404
+
+from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.es.domains import DomainES
+from corehq.apps.es.apps import AppES
+
+
+logger = logging.getLogger('app_migration')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    help = '''
+        Identify forms that were migrated because they had preload values,
+        even though they did not require a case, which breaks the form.
+    '''
+
+    def handle(self, *args, **options):
+        # Find all apps using vellum case management
+        app_query = AppES().is_build(False).term('vellum_case_management', True).source(['domain', '_id'])
+
+        # Find all domains created after vellum case management was released,
+        # since their apps were never in the old world and therefore never migrated
+        epoch = datetime(year=2016, month=7, day=15)
```

Comment:
Is this the day cmitfb was release to new domains? It would be better to do the day after as it's possible that someone created a domain in the morning that was old world and ES begins dates at midnight